Cannot build source; `base64` is ambiguous

Build fails on Ubuntu 20.04.6 system due to ambiguous import.
```
...
Compiling gateway-rs v1.2.1 (/home/ubuntu/gateway-rs)
error[E0659]: `base64` is ambiguous
  --> src/lib.rs:20:16
   |
20 | pub(crate) use base64::*;
   |                ^^^^^^ ambiguous name
   |
   = note: ambiguous because of multiple potential import sources
   = note: `base64` could refer to a crate passed with `--extern`
   = help: use `::base64` to refer to this crate unambiguously
note: `base64` could also refer to the module defined here
  --> src/lib.rs:18:1
   |
18 | mod base64;
   | ^^^^^^^^^^^
   = help: use `crate::base64` to refer to this module unambiguously

For more information about this error, try `rustc --explain E0659`.
error: could not compile `gateway-rs` (lib) due to previous error```

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -17,7 +17,7 @@ pub mod sync;
 mod api;
 mod base64;
 
-pub(crate) use base64::*;
+pub(crate) use crate::base64::Base64;
 pub use beacon::{Region, RegionParams};
 pub use error::{Error, Result};
 pub use keyed_uri::KeyedUri;

